scripts/dts/new_extract.py

```diff
@@ -55,26 +55,39 @@ def main():
             print("#define DT_SRAM_BASE_ADDRESS\t"
                       + hex(edt.sram_dev.regs[0].addr),
                   file=out)
+            print("#define DT_SRAM_SIZE\t"
+                      + ascii(int(edt.sram_dev.regs[0].size / 1024)),
```

Comment:
Can do

```python
print("#define DT_SRAM_SIZE\t" + str(edt.sram_dev.regs[0].size//1024), file=out)
```

or

```python
print("#define DT_SRAM_SIZE\t{}".format(edt.sram_dev.regs[0].size//1024), file=out)
```

`//` means integer (truncating) division. With `/`, you get a float.

Reply:
Can simplify by passing multiple arguments to `print()` too:

```python
print("#define DT_SRAM_SIZE\t", edt.sram_dev.regs[0].size//1024, file=out)
```